8.0/root/usr/share/container-scripts/mysql/README.md

```diff
@@ -187,16 +188,26 @@ location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by settin
 
 Extending image
 ---------------
-This image can be extended using [source-to-image](https://github.com/openshift/source-to-image).
+This image can be extended in Openshift using the `Source` build strategy or via the standalone
+[source-to-image](https://github.com/openshift/source-to-image) application (where available).
+For this, we will assume that you are using the `rhscl/mysql-80-rhel7` image,
+available via `mysql:8.0` imagestream tag in Openshift.
 
-For example, to build a customized MariaDB database image `my-mysql-rhel7`
-with a configuration in `~/image-configuration/` run:
+
+For example, to build a customized MySQL database image `my-mysql-rhel7`
+with a configuration in `image-configuration/` run:
+
+```
+$ oc new-app mysql:8.0~./image-configuration/ --name my-mysql-rhel7
+```
+
+or via s2i:
 
 ```
-$ s2i build ~/image-configuration/ rhscl/mysql-80-rhel7 my-mysql-rhel7
+$ s2i build ./image-configuration/ rhscl/mysql-80-rhel7 new-postgresql
```

Comment:
Probably mistake

Reply:
it is in mariadb container too

Reply:
Definitely a mistake, will fix in both. Thanks for the review!